projects/frontegg-app/src/lib/frontegg-app.service.ts

```diff
@@ -29,6 +29,7 @@ export class FronteggAppService {
     isLoading: true,
     isAuthenticated: false,
   } as FronteggState['auth']);
+  private entitlementsStateSubject = new BehaviorSubject<any>(null);
```

Comment:
why any?

Reply:
Entitlements | null

Reply:
it's not entitlements.. it's a type I have inside the internal logic.. I'm thinking how to handle it :/ 
lemme know if you have an idea